dotnet/test/Microsoft.AutoGen.Integration.Tests/HelloAppHostIntegrationTests.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// HelloAppHostIntegrationTests.cs
+
+using System.Text.Json;
+using Xunit.Abstractions;
+
+namespace Microsoft.AutoGen.Integration.Tests;
+
+public class HelloAppHostIntegrationTests(ITestOutputHelper testOutput)
+{
+    [Theory, Trait("type", "integration")]
+    [MemberData(nameof(AppHostAssemblies))]
+    public async Task AppHostRunsCleanly(string appHostPath)
+    {
+        var appHost = await DistributedApplicationTestFactory.CreateAsync(appHostPath, testOutput);
+        await using var app = await appHost.BuildAsync().WaitAsync(TimeSpan.FromSeconds(15));
+
+        await app.StartAsync().WaitAsync(TimeSpan.FromSeconds(120));
+        await app.WaitForResourcesAsync().WaitAsync(TimeSpan.FromSeconds(120));
+
+        app.EnsureNoErrorsLogged();
+        await app.StopAsync().WaitAsync(TimeSpan.FromSeconds(15));
+    }
```

Comment:
Can we add verification on exiting log as well after `app.StopAsync` to ensure the app terminate correctly

Reply:
https://github.com/microsoft/autogen/issues/4619